test/libsolidity/ASTLegacyJSON.cpp

```diff
@@ -266,15 +266,13 @@ BOOST_AUTO_TEST_CASE(function_type)
 	BOOST_CHECK_EQUAL(argument["attributes"]["name"], "x");
 	BOOST_CHECK_EQUAL(argument["attributes"]["type"], "function () payable external returns (uint256)");
 	Json::Value funType = argument["children"][0];
-	BOOST_CHECK_EQUAL(funType["attributes"]["constant"], false);
```

Comment:
Can you add the corresponding check about stateMutability?

Reply:
Done.